fe/fe-core/src/main/java/com/starrocks/sql/analyzer/FunctionAnalyzer.java

```diff
@@ -218,26 +229,23 @@ private static void analyzeBuiltinAggFunction(FunctionCallExpr functionCallExpr)
             return;
         }
 
-        if ((fnName.getFunction().equalsIgnoreCase("HLL_UNION_AGG")
-                || fnName.getFunction().equalsIgnoreCase("HLL_UNION")
-                || fnName.getFunction().equalsIgnoreCase("HLL_CARDINALITY")
-                || fnName.getFunction().equalsIgnoreCase("HLL_RAW_AGG"))
+        if ((fnName.getFunction().equals(FunctionSet.HLL_UNION_AGG)
+                || fnName.getFunction().equals(FunctionSet.HLL_UNION)
+                || fnName.getFunction().equals(FunctionSet.HLL_CARDINALITY)
+                || fnName.getFunction().equals(FunctionSet.HLL_RAW_AGG))
                 && !arg.getType().isHllType()) {
             throw new SemanticException(
                     "HLL_UNION_AGG, HLL_RAW_AGG and HLL_CARDINALITY's params must be hll column");
         }
 
-        if (fnName.getFunction().equalsIgnoreCase("min")
-                || fnName.getFunction().equalsIgnoreCase("max")) {
+        if (fnName.getFunction().equals(FunctionSet.MIN)
+                || fnName.getFunction().equals(FunctionSet.MAX)
+                || fnName.getFunction().equals(FunctionSet.NDV)
+                || fnName.getFunction().equals(FunctionSet.HLL_UNION_AGG)) {
             fnParams.setIsDistinct(false);  // DISTINCT is meaningless here
-        } else if (fnName.getFunction().equalsIgnoreCase("DISTINCT_PC")
```

Comment:
what is `DISTINCT_PC` ?

Reply:
I don't know,  but this function isn’t implemented always，so I remove  it.